src/app/view/components/login/dot-login-component/dot-login.component.ts

```diff
@@ -0,0 +1,181 @@
+import {
+    Component,
+    EventEmitter,
+    Input,
+    Output,
+    AfterViewInit,
+    OnInit,
+    ViewChild,
+    ElementRef
+} from '@angular/core';
+import { LoginService, LoggerService } from 'dotcms-js';
+import { DotLoginData, DotLoginLanguage, DotSystemInformation } from '@models/dot-login';
+import { SelectItem } from 'primeng/api';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+    selector: 'dot-login-component',
+    templateUrl: './dot-login.component.html',
+    styleUrls: ['./dot-login.component.scss']
+})
+/**
+ * The login component allows the user to fill all
+ * the info required to log in the dotCMS angular backend
+ */
+export class DotLoginComponent implements AfterViewInit, OnInit {
+    @Input() message = '';
+    @Input() passwordChanged = false;
```

Comment:
why this one is an input?

we just use it for set a message here (I think):

https://github.com/dotCMS/core-web/pull/956/files#diff-de2406f4bdb3f68d7acf0c0e8a2f34c6R135

but message is a inout also so we could just set message and remove 'passwordChanged' input

same for resetEmailSent